doc/rados/configuration/mon-osd-interaction.rst

```diff
@@ -31,8 +31,9 @@ or by setting the value at runtime. If a neighboring Ceph OSD Daemon doesn't
 show a heartbeat within a 20 second grace period, the Ceph OSD Daemon may
 consider the neighboring Ceph OSD Daemon ``down`` and report it back to a Ceph
 Monitor, which will update the Ceph Cluster Map. You may change this grace
-period by adding an ``osd heartbeat grace`` setting under the ``[osd]`` section
-of your Ceph configuration file, or by setting the value at runtime.
+period by adding an ``osd heartbeat grace`` setting under the ```[mon]```
+and ``[osd]`` section of your Ceph configuration file, or by setting the
```

Comment:
setting in `[global]` will apply to both osds and mons, btw. Maybe worth mentioning that as well. Same for the `description` section in the other chunk of the patch.